chia/_tests/pools/test_pool_cmdline.py

```diff
@@ -885,11 +891,14 @@ async def test_plotnft_cli_change_payout(
 ) -> None:
     wallet_state_manager: WalletStateManager = wallet_environments.environments[0].wallet_state_manager
     wallet_rpc: WalletRpcClient = wallet_environments.environments[0].rpc_client
-    _client_info: WalletClientInfo = WalletClientInfo(
+    client_info: WalletClientInfo = WalletClientInfo(
         wallet_rpc,
         wallet_state_manager.root_pubkey.get_fingerprint(),
         wallet_state_manager.config,
     )
+    wallet_state_manager.config["reuse_public_key_for_change"][str(client_info.fingerprint)] = (
+        wallet_environments.tx_config.reuse_puzhash
+    )
```

Comment:
If we're going to do this, we at least need a comment saying it's a hack that makes up for the lack of the ability to specify proper tx options at the command line level.

Reply:
This appears true, but the plotnft code has specific flows for updating the config that happen well outside the TX boundary that need to check this config item as well - so either way the config needs to be matched up